src/PKSim.Infrastructure/ORM/DAS/DAS.cs

```diff
@@ -508,10 +515,36 @@ public void FillDataTable(DASDataTable dataTable, string sql)
          dataTable.Rows.Clear();
          try
          {
-            using (var adapter = _providerFactory.CreateDataAdapter())
+            // Manual data reading to avoid constraint issues with DataTable.Load()
```

Comment:
This change is a bit of a mystery to me. Are there some constraints that were ignored before

Reply:
What is this code doing in the first place?

Reply:
dataTable.Load method
<img width="564" height="128" alt="image" src="https://github.com/user-attachments/assets/699391ea-b117-47d1-befa-10e40576641f" />

Reply:
It's because the constraints are built based on the underlying tables. In this example `tab_species` does have a unique key on the species column since it's the primary key for that table.

The manual update code does not analyze the schema and so does not have this problem. It's also possible that the System.Data.SQLite also could not determine the underlying tables.